contrib/cirrus/lib.sh

```diff
@@ -192,10 +192,13 @@ in_podman() {
         envargs+=("-e" "$var")
     done <<<"$(passthrough_envars)"
 
+    mount -t tmpfs -o size=75%,mode=0700 none /tmp
```

Comment:
This seems dangerous, why would we over mount /tmp on an existing running system. What if another application needs access to files it already wrote there.

Second mode=0700 seems just not suitable for /tmp that has to be writeable for everybody